.github/workflows/dotnet-core-cov.yml

```diff
@@ -0,0 +1,41 @@
+name: .NET Code Coverage
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
we are enabling codecoverage just in linux?

Reply:
i was checking for ubuntu how that would work. i'm creating a new pr enabling for windows as well